lib/ingestors/ingestor_event_rest.rb

```diff
@@ -745,9 +745,11 @@ def process_wur(url)
             when 'provider'
               event.organizer = element.text
             when 'startdate', 'courseDate'
-              event.start = element.text.to_s.to_time
+              event.start = element.text.to_s
+              event.start = (event.start.split[0, 4] + event.start.split[-3, 1]).join(' ').to_time
             when 'enddate', 'courseEndDate'
-              event.end = element.text.to_s.to_time
+              event.end = element.text.to_s
+              event.end = (event.end.split[0, 4] + event.end.split[-1, 1]).join(' ').to_time
```

Comment:
to be honest this looks quite fragile. Can you extract this part into a separate method and test it for the most common cases you encounter currently on the WUR site?